docs/source/msp.rst

```diff
@@ -189,17 +189,22 @@ below:
 
    NodeOUs:
      Enable: true
+     # If a specific Certificate is not configured per identifier below,
+     # then the identity classification OU can be issued by any of the organization's configured CAs.
+     # This allows for adding or rotating CA or intermediate certs later,
+     # without having to reissue all credentials.
+     # Configure a Certificate only if you want a specific CA or intermediate cert to issue identity classification OUs.
```

Comment:
I'm unclear on the use case. Can you provide an example of what you mean? 
It seems like you are suggesting to not specify an OrganizationalUnitIdentifer for an identity? Sort of like leave it empty for now?

Reply:
@pamandrejko The general use case is included in the comment and in the rst sentence below...

General Use Case: I want to be able to add or rotate my organization's CA or intermediate certs later, without having to replace all the existing client/admin/peer/orderer credentials at the same time.

This is not about "OrganizationalUnitIdentifier". It is about not configuring a Certificate per OU identifier. See the commented out Certificate lines below. The problem is that only a single Certificate can be configured, not multiple. So by configuring a single certificate here, it will render node OU classifications issued from other CA or intermediate certs from the same org invalid.

This came up in a side-by-side migration scenarios, where a second CA and second set of peers is expected to come online, while the original peers from the original CA are expected to keep working. To keep both sets working, an individual certificate must not be configured.  (It would have been nice to be able to configure multiple certificates, but that would require a capability change...).

With this knowledge, what do you recommend be added to make it more clear?

Reply:
Thanks, I addressed the comments.